models/inventory/StockTransfer.ts

```diff
@@ -246,6 +270,30 @@ export abstract class StockTransfer extends Transfer {
     await invoice.setAndSync('stockNotTransferred', notTransferred);
   }
 
+  async _updateItemsReturned() {
+    if (!this.returnAgainst) {
+      return null;
+    }
+
+    const returnDocs = await this.fyo.db.getAll(this.schema.name, {
+      filters: {
+        returnAgainst: this.returnAgainst,
+        submitted: true,
+        cancelled: false,
+      },
+    });
+
+    const isItemsReturned = returnDocs.length;
```

Comment:
Variable names prefixed with `is` imply a `boolean`, don't break norms such as this, either convert the value into boolean or call it something else.

Reply:
corrected